src/palette.ml

```diff
@@ -110,6 +110,31 @@ let load_tic80_palette (raw : string) : t =
   else
     raise (Invalid_argument "Palette size must not be zero or negative")
 
+(* New function for Task #27 *)
```

Comment:
Can you remove this comment - it's fine whilst you're working on the PR, but just adds noise later.